web-ui/src/components/member-directory/AdminMemberCard.jsx

```diff
@@ -225,6 +234,12 @@ const AdminMemberCard = ({ member, index }) => {
                     type: UPDATE_MEMBER_PROFILES,
                     payload: copy
                   });
+                  if (oldMember && oldMember.pdlId !== member.pdlId) {
+                    await emailPDLAssignment(member, csrf)
+                  }
+                  if (oldMember && oldMember.supervisorid !== member.supervisorid) {
+                    await emailSupervisorAssignment(member, csrf)
+                  }
```

Comment:
Do we need to await these email responses before proceeding if we're not doing anything with the response? Unless we want to log error messages when these fail, I think we could just fire them like:

```js
if (oldMember) {
  oldMember.pdlId !== member.pdlId && emailPDLAssignment(member, csrf);
  oldMember.supervisorid !== member.supervisorid && emailSupervisorAssignment(member, csrf);
}
```

This will prevent blocking the thread before `handleClose` is called, allowing the dialog to close more quickly.

Reply:
Oh for some reason didnt see your code comment. Looking at that now.

Reply:
Added